hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1128,6 +1128,9 @@
 
   /** Conf key for enabling meta replication */
   public static final String USE_META_REPLICAS = "hbase.meta.replicas.use";
+  public static final String META_REPLICAS_MODE = "hbase.meta.replicas.mode";
+  public static final String META_REPLICAS_MODE_LOADBALANCE_REPILCA_CHOOSER =
+    "hbase.meta.replicas.mode.loadbalance.replica.chooser";
```

Comment:
I see you're forced to work with existing configuration names, but i wish these were all `hbase.meta_replica.`